src/repository.c

```diff
@@ -1066,25 +1066,42 @@ void git_repository_set_config(git_repository *repo, git_config *config)
 	set_config(repo, config);
 }
 
-int git_repository_odb__weakptr(git_odb **out, git_repository *repo)
+static int git_repository__setup_odb(git_odb **out, git_repository *repo)
```

Comment:
We prefer to not have `git_whatever__` prefixes for static functions which are local to the compilation unit only.